packages/asset-swapper/test/swap_quote_consumer_test.ts

```diff
@@ -9,7 +10,6 @@ import { SwapQuote, SwapQuoteConsumer } from '../src';
 import { ExtensionContractType } from '../src/types';
 
 import { chaiSetup } from './utils/chai_setup';
-import { migrateOnceAsync } from './utils/migrate';
```

Comment:
Why were changes introduced to `asset-swapper`?

Reply:
`migrateOnceAsync` was identical in two files and the unit tests here also needed it. So instead of making a third file I moved it into `@0x/migrations`